target-list-with-crowdsec/create-target-list.sh

```diff
@@ -11,14 +11,14 @@ banned_ips=$(curl --request GET \
   | jq -r '.[] | .ip')
 
 json_payload='{
-  "name": "A Simple Target List with crowdsec",
+  "name": "Target List with crowdsec",
   "notes":"This is a Simple Target List with crowdsec",
   "targets": ['
 json_payload+=$(echo "$banned_ips" | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/","/g')
 json_payload+=']
 }'
 
-http_code=$(curl -s -w %{http_code} --request POST \
+http_code=$(curl -s -w %{http_code} -o /dev/null --request POST \
```

Comment:
I assume the `<Firewalla Domain>` part will be replaced somewhere before this script can be executed, right ?